modules/core/src/main/java/org/apache/ignite/configuration/ClientConfiguration.java

```diff
@@ -115,6 +118,9 @@
     /** Reconnect throttling retries. See {@code reconnectThrottlingPeriod}. */
     private int reconnectThrottlingRetries = 3;
 
+    /** Executor for async operations continuations. */
+    private Executor asyncContinuationExecutor;
```

Comment:
Is it `null` by default? Does it mean, user will get an exception in runtime if it uses continuation with no executor set?

Maybe, we should set some implementation here as a default value? I'm pretty much sure, it is going to be fine for the most users, at least at the beginning of the development.

Reply:
This is a private property, please see public setter and getter, there is a detailed javadoc.